packages/react-environment/src/api-providers/web3/web3-bridge-withdraw.ts

```diff
@@ -214,29 +298,12 @@ export class Web3BridgeWithdraw extends BridgeWithdraw<WebbWeb3Provider> {
     const targetChainName = getEVMChainNameFromInternal(Number(depositNote.chain) as ChainId);
     logger.trace(`Bridge withdraw from ${sourceChainName} to ${targetChainName}`);
 
-    const token = parseNote.note.tokenSymbol;
-    const bridgeCurrency = BridgeCurrency.fromString(token);
-    const bridge = Bridge.from(this.bridgeConfig, bridgeCurrency);
-
-    try {
-      if (depositNote.sourceChain === depositNote.chain) {
-        logger.trace(`Same chain flow ${sourceChainName}`);
-        return this.sameChainWithdraw(depositNote, recipient);
-      } else {
-        logger.trace(`cross chain flow ${sourceChainName} ----> ${targetChainName}`);
-        return this.crossChainWithdraw(depositNote, recipient);
-      }
-    } catch (e) {
-      this.emit('stateChange', WithdrawState.Ideal);
-      transactionNotificationConfig.failed?.({
-        address: recipient,
-        data: e?.code === 40001 ? 'Withdraw rejected' : 'Withdraw failed',
-        key: 'bridge-withdraw-evm',
-        path: {
-          method: 'withdraw',
-          section: `Bridge ${bridge.currency.chainIds.map(getEVMChainNameFromInternal).join('-')}`,
-        },
-      });
```

Comment:
This try/catch does not work for some reason. Not sure how we can remove duplicate code for metamask rejection throughout these files.